docs/index.html

```diff
@@ -7,6 +7,172 @@
+    }
+    /* ---- nav: logo image ----- */
+    .evTpW {
+      padding: 1em 16px;
+    }
+    /* ---- nav text: primary items ----- */
+    .iTiWpH, .dttBLN {
+      font-size: 1.1rem;
+      font-weight: 700;
+    }
+    /* ---- nav text: secondary items ----- */
+    .hcwWID, .jBjYkL, .idRuNa, .hxqNKs {
+      font-size: 0.9rem;
+      font-weight: 600;
+    }
+    /* ---- nav text: active items ----- */
+    .hxqNKs, .dttBLN, .jBjYkL {
+      border-left: 4px solid #3e9096;
+      background-color: #fff;
+    }
+    .hxqNKs > .jzElRo, .dttBLN > .jzElRo, .jBjYkL > .jzElRo {
+      color: #3e9096;
+    }
+    /* ---- nav text: active item hover ----- */
+    .hxqNKs:hover, .dttBLN:hover, .jBjYkL:hover {
+      color: #3e9096;
+      background-color: #fff;
+    }
+    /* ---- nav text: inactive item hover ----- */
+    .idRuNa:hover, .iTiWpH:hover, .hcwWID:hover {
+      color: #3e9096;
+      background-color: #f7f8fa;
+    }
+    /* ---- nav badges ----- */
+    .hbxWyq {
+      font-size: 10px;
+      margin-top: 7px;
+    }
+    /* ---- nav text: ReDoc link ----- */
+    .bmRLPL a {
+      text-decoration: none;
+      color: #d9dbe2;
+    }
+    /* ---- running text: inline <code> ----- */
+    .dCnGCn code, .bMfIUD, .beUper {
+      color: rgba(51,51,51,.8);
+      padding: 3px 6px;
+      margin: 0;
+      background-color: rgba(27,31,35,.05);
+      border-radius: 3px;
+      border: 0px;
+      font-size: 1em;
+    }
+    /* ---- running text: decrease section padding ----- */
+    .cncswi {
+      padding: 1em 0;
+    }
+    /* ---- running text: H{x} in heavier weight ----- */
+    .WxWXp, .givSXQ, .ioYTqA, .kBiOhJ {
+      font-weight: 600;
+    }
+    /* ---- running text: Main title ----- */
+    .givSXQ {
+      font-size: 2rem;
+    }
+    /* ---- running text: H2 ----- */
+    .ioYTqA {
+      margin: 0;
+    }
+    /* ---- running text: responses expand/collapse and 'required' legend ----- */
+    .dUFXgZ, .bGITIP {
+      color: #ea5037;
+    }
+    .dUFXgZ:focus {
+      outline: #ea5037 auto;
+    }
+    /* ---- running text: regular buttons ----- */
+    .jrowNk {
+      background-color: #2b6569;
+      border: 0px;
+      border-radius: 2px;
+      color: #fff;
+      font-weight: 600;
+    }
+    .jrowNk:hover {
+      background-color: #439a9d;
+    }
+    /* ---- try-out area: background ----- */
+    .cObJOV, .celZWI {
+      background-color: #edf0f4;
+    }
+    /* ---- try-out area: H3 ----- */
+    .kBWwoV {
+      color: #202020;
+    }
+    /* ---- try-out area: drop-downs ----- */
+    .gLTsd, .iGGYCX {
+      background-color: #7f8491;
+    }
+    .gLTsd:focus, .iGGYCX:focus {
+      -webkit-box-shadow: none;
+      -moz-box-shadow: none;
+      box-shadow: none;
+    }
+    /* ---- try-out area: GET/POST/DEL font weight ----- */
+    .eassAJ, .hjuNDJ, .eTPsXv {
+      font-weight: 600;
+    }
+    /* ---- try-out area: buttons ----- */
+    .dOwpAS > ul > li {
+      background-color: #b7bbc8;
```

Comment:
It may be just my setup, but the contrast on those buttons was a bit aggressive, would be it ok to change:

```suggestion
      background-color: transparent;
```

Before:

> ![2020-07-23--22-20-56](https://user-images.githubusercontent.com/157609/88335688-01c67c00-cd34-11ea-9e58-5bb4085ff828.png)

After:

>  ![2020-07-23--22-29-34](https://user-images.githubusercontent.com/157609/88335685-012de580-cd34-11ea-9c98-67bc1a16f202.png)

Reply:
@lidel - the styling was intended to match docs.ipfs.io as closely as possible. Are you seeing a major discrepancy in linux?

Reply:
Ah, yes, they look pretty similar. Ignore my comment about heavy headers then (i was comparing with the old version).
In that case the only thing I have problem with is the contrast of those buttons above.

Reply:
@lidel sounds good - I can do it in a bit or if you want to sneak in there, feel free.

Reply:
@lidel sounds good - committed your suggestion, please feel free to merge PR at your leisure.